exercises/resistor-color/test/test_resistor_color.c

```diff
@@ -0,0 +1,49 @@
+#include "vendor/unity.h"
+#include "../src/resistor_color.h"
+
+void setUp(void)
+{
+}
+
+void tearDown(void)
+{
+}
+
+static void test_black(void)
+{
+   TEST_ASSERT_EQUAL_UINT16(0, colorCode(BLACK));
+}
+
+static void test_white(void)
+{
+   TEST_IGNORE();               // delete this line to run test
+   TEST_ASSERT_EQUAL_UINT16(9, colorCode(WHITE));
+}
+
+static void test_orange(void)
+{
+   TEST_IGNORE();
+   TEST_ASSERT_EQUAL_UINT16(3, colorCode(ORANGE));
+}
+
+static void test_colors(void)
+{
+   TEST_IGNORE();
+   const resistor_band_t expected[] = {
+      BLACK, BROWN, RED, ORANGE, YELLOW,
+      GREEN, BLUE, VIOLET, GREY, WHITE
+   };
+   TEST_ASSERT_EQUAL_INT_ARRAY(expected, colors(), COLOR_COUNT);
```

Comment:
> dependent upon the user's solution in a non-behavioral way (this can be way wrong and the module will otherwise work correctly) and is dependent upon the user's solution in a way that can mask problems because the solution feeds into the checks instead of the checks being independent.

True dat!